be/src/runtime/data_stream_recvr.cc

```diff
@@ -176,14 +176,12 @@ Status DataStreamRecvr::SenderQueue::get_batch(RowBatch** next_batch) {
     *next_batch = _current_batch.get();
 
     if (!_pending_closures.empty()) {
-        auto done = _pending_closures.front();
-        done->Run();
+        auto closure_pair = _pending_closures.front();
+        closure_pair.first->Run();
```

Comment:
Seems have not call start() of MonotonicStopWatch

Reply:
![image](https://user-images.githubusercontent.com/10553413/85491848-d0f50900-b606-11ea-9ecb-8fef87b44750.png)
@chaoyli Hi，the MonotonicStopWatch start here.